src/Event/FastEvmProxy.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Penny\Event;
+
+use FastEventManager\EventManager;
+
+class FastEvmProxy implements EventManagerInterface
```

Comment:
no, there are other proxies that works for zf evm, cake evm too

Warm regards,

Abdul Malik Ikhsan

Pada 2 Feb 2016, pukul 18.55, Simone <notifications@github.com> menulis:

> In src/Event/FastEvmProxy.php:
> 
> > @@ -0,0 +1,38 @@
> > +<?php
> > +namespace Penny\Event;
> > +
> > +use FastEventManager\EventManager;
> > +
> > +class FastEvmProxy implements EventManagerInterface
> This is just a proxy in Penny domain, but it is called FastEvmProxy. IMHO Proxy or EventProxy should be enough.
> 
> —
> Reply to this email directly or view it on GitHub.
>